src/content/developers/tutorials/how-to-mint-an-nft.md

```diff
@@ -0,0 +1,275 @@
+
+# How to Mint an NFT (Part 2/3 of NFT Tutorial Series)
+
+This tutorial describes how to mint an NFT on the Ethereum blockchain using our smart contract and Web3. You should have already completed Part 1: How to Write & Deploy an NFT, prior to starting Part 2 below. In Part 3, we will explain how to view your freshly minted NFT on Metamask!
+
+##
+
+[Beeple](https://www.wsj.com/articles/beeple-nft-fetches-record-breaking-69-million-in-christies-sale-11615477732): $69 Million
+[3LAU](https://www.forbes.com/sites/abrambrown/2021/03/03/3lau-nft-nonfungible-tokens-justin-blau/?sh=5f72ef64643b): $11 Million
+[Grimes](https://www.theguardian.com/music/2021/mar/02/grimes-sells-digital-art-collection-non-fungible-tokens): $6 Million
+
+All of them minted their NFT’s using Alchemy’s powerful API. In this tutorial, we’ll teach you how to the same in <10 minutes.
+
+“Minting a NFT” is the act of publishing a unique instance of your ERC721 token on the blockchain. Using our smart contract from Part 1 of this NFT tutorial series, let’s flex our web3 skills and mint a NFT. At the end of this tutorial, you’ll be able to mint as many NFTs as your heart (and wallet) desires!
+
+Let’s get started! 🪄
+
+## Step 1: Install web3
+
+If you followed the first tutorial on creating your NFT smart contract, you already have experience using Ethers.js. Web3 is similar to Ethers, as it is a library used to make creating requests to the Ethereum blockchain easier. In this tutorial we’ll be using [Alchemy Web3](https://docs.alchemyapi.io/alchemy/documentation/alchemy-web3), which is an enhanced web3 library that offers automatic retries and robust WebSocket support.
+
+In your project home directory run:
+
+    npm install @alch/alchemy-web3
```

Comment:
reminder that this site requires backticks to denote inline code, and triple backticks for code blocks, not just indentation (please update throughout)